sim/shaman/item_sets_phase_4.go

```diff
@@ -0,0 +1,307 @@
+package shaman
```

Comment:
With these changes we're going to be breaking out a lot of functions so I think it would be good to separate PvE sets by phase at least starting with Phase 4